src/api/spec/models/workflow/step/configure_repositories_spec.rb

```diff
@@ -301,87 +298,7 @@
       end
 
       it 'raises an error due to an inexistent target project' do
-        expect { subject.call }.to raise_error(Project::Errors::UnknownObjectError, "Project not found: #{target_project_name}")
-      end
-    end
-  end
-
-  describe '#target_project_name' do
```

Comment:
With the removal of those specs, the method isn't tested anywhere. We should instead keep those specs, but move them to `src/api/spec/models/workflow/step_spec.rb` (this file doesn't exist yet)

Reply:
done